core/src/packet_forward_manager.rs

```diff
@@ -0,0 +1,427 @@
+use {
```

Comment:
It seems like all the forwarding changes can also be broken into a separate PR after the main prioritization change lands